Financial-Times/n-myft-dropdown
# ug-1017 bugfix: safari styling

### Description
There is currently a bug that affects styling in Safari desktop - this PR is to fix this issue.

### Ticket
[ug-1017](https://financialtimes.atlassian.net/browse/UG-1017)

### Screenshots

**Before**
<img width="1238" alt="Screenshot 2022-10-14 at 15 07 28" src="https://user-images.githubusercontent.com/54366961/195867199-a9e0c14a-2f84-4d32-9727-37bba79b88d0.png">
**After**
<img width="1236" alt="Screenshot 2022-10-14 at 15 08 04" src="https://user-images.githubusercontent.com/54366961/195867401-31d21667-c9f1-4077-b025-e145f1225fbb.png">   

### How do I test this code?
1. Checkout this branch locally and run `npm link`
2. In `next-article` run `npm link @financial-times/n-myft-dropdown`
3. Run next article locally and load an article, e.g. https://local.ft.com:5050/content/99eafd52-eceb-44bf-ac1c-ba9a71913cec in Safari. Ensure that [myFTDropdownTest](https://toggler.ft.com/#myFTDropdownTest) is set to 'variant'. Check that the menu appears (and opens) on the right hand side, as it currently does in prod on Chrome/Firefox. Check mobile view has not been affected.
4. Load the article on other browsers - desktop and mobile - to ensure that it hasn't impacted any styling (each browser needs the flag setting in toggler).
5. Repeat steps 2-4 with `next-home-page`, loading http://localhost:3000 (you may need to alter next-home-page's [setup.ts](https://github.com/Financial-Times/next-home-page/blob/f5519646d53ee8297148890e5e6657b6c7dd3fb2/apps/home-page/client/setup.ts#L103-L105) to initialise the component without a flag as toggler doesn't work on localhost).

### Reminder
Have you completed these common tasks? (remove those that don't apply)

- [x] **Guidance for reviewer** communicated expected changes / behaviour

### How we will review this PR
Our team uses [conventional comments](https://conventionalcomments.org/) to provide feedback.
